bundles/CustomReportsBundle/src/Tool/Adapter/Sql.php

```diff
@@ -46,11 +58,90 @@ public function getData(?array $filters, ?string $sort, ?string $dir, ?int $offs
             }
 
             $data = $db->fetchAllAssociative($sql);
+
+            $this->loadRelationData($data);
         }
 
         return ['data' => $data, 'total' => $total];
     }
 
+    protected function loadRelationData(array &$data): void
+    {
+        $columnsDictionary = $this->getRelationColumns();
```

Comment:
if this is empty, we could skip the whole function, right?

Reply:
@gjorgic :ping_pong:

Reply:
please, can you check now